js_modules/dagit/packages/core/src/assets/LaunchAssetExecutionButton.tsx

```diff
@@ -186,24 +196,33 @@ export const LaunchAssetExecutionButton: React.FC<{
           position="bottom-right"
           useDisabledButtonTooltipFix
         >
-          <MaterializeButton
-            intent={intent}
-            data-testid={testId('materialize-button')}
-            onClick={(e) => onClick(firstOption.assetKeys, e)}
-            style={
-              options.length > 1
-                ? {
-                    borderTopRightRadius: 0,
-                    borderBottomRightRadius: 0,
-                    borderRight: `1px solid rgba(255,255,255,0.2)`,
-                  }
-                : {}
-            }
-            disabled={!firstOption.assetKeys.length}
-            icon={loading ? <Spinner purpose="body-text" /> : <Icon name="materialization" />}
-          >
-            {firstOption.label}
-          </MaterializeButton>
+          <JoinedButtons>
+            <MaterializeButton
+              intent={intent}
+              data-testid={testId('materialize-button')}
+              onClick={(e) => onClick(firstOption.assetKeys, e)}
+              style={
+                options.length > 1
+                  ? {
+                      borderTopRightRadius: 0,
+                      borderBottomRightRadius: 0,
+                      borderRight: `1px solid rgba(255,255,255,0.2)`,
+                    }
+                  : {}
+              }
+              disabled={!firstOption.assetKeys.length}
+              icon={loading ? <Spinner purpose="body-text" /> : <Icon name="materialization" />}
+            >
+              {firstOption.label}
+            </MaterializeButton>
+            <MaterialzeButtonOptions
+              openWithLaunchpad={(e: React.MouseEvent<any>) => {
+                e.shiftKey = true;
+                onClick(firstOption.assetKeys, e);
```

Comment:
Weird but seems fine to me since it's a fairly obvious thing. Might be good to double check that shiftKey is writable in all browser though.